dxr/plugins/python/tests/test_menus/test_menus.py

```diff
@@ -0,0 +1,70 @@
+from textwrap import dedent
+from urllib import quote
+
+from dxr.plugins.python.tests import PythonSingleFileTestCase
+from dxr.testing import DxrInstanceTestCase, menu_on
+
+
+def search(query):
+    return '/code/search?q=' + quote(query)
+
+
+def browse(path, lineno=None):
+    return '/code/source/' + path + (('#' + unicode(lineno)) if lineno else '')
```

Comment:
This will sometimes return unicode and other times str. It works out fine so far, but let's pick one.